src/modules/userCreationRequestMethods.ts

```diff
@@ -787,11 +902,13 @@ export const userCreationRequestsMethods = {
             workSpace: workSpace || undefined,
             personalEmail: personalEmail || undefined,
             workEmail: workEmail || undefined,
-            supplementalPositions: supplementalPositions.map(({ organizationUnitId, unitId, percentage }) => ({
-                organizationUnitId,
-                unitId: unitId || undefined,
-                percentage,
-            })),
+            supplementalPositions: supplementalPositions
+                .filter((position) => !position.main)
```

Comment:
main position is not returned?

Reply:
this field is only for additional position, main goes in organizationUnitId